### PR TITLE
Add MAYO

### PR DESCRIPTION
**Pull request by bhess**
Tracker for MAYO integration in liboqs: https://github.com/open-quantum-safe/liboqs/pull/1707

### CODE REVIEW

**Review by Gangstablue777 — COMMENTED** [review]
I thought  he did ok job

**Comment by baentsch** [comment]
CI for this PR queued for more than 20h: 
grafik

This does [not seem to be a transient GH problem as per https://www.githubstatus.com/](https://www.githubstatus.com/), so what's going on here? Should we simply reduce MacOS CI/support or is this sth that can be resolved with proper funding?  @ryjones ?

**Comment by bhess** [comment]
> CI for this PR queued for more than 20h: grafik
> 
> This does [not seem to be a transient GH problem as per https://www.githubstatus.com/](https://www.githubstatus.com/), so what's going on here? Should we simply reduce MacOS CI/support or is this sth that can be resolved with proper funding? @ryjones ?

I don't know if this is related to the CI not running, but MacOS 11 seems to be end of life: https://endoflife.date/macos, so I think it should be ok to drop the macos-11 runner (done now in this PR).

**Comment by ryjones** [comment]
Yes, @bhess @baentsch , GitHub has dropped MacOS 11 support.

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
         os:
           - macos-13
           - macos-12
-          - macos-11
+          #- macos-11 // Runner not online
```
Comment (baentsch) [inline .github/workflows/macos.yml:18]: Please remove for good

**Review by baentsch — APPROVED** [review]

File: .github/workflows/macos.yml (hunk 1 above)
Comment (bhess) [reply .github/workflows/macos.yml]: Done

**Pull request merged**